### PR TITLE
chore: update trustbloc-did-method dependency

### PR DESCRIPTION
**Pull request by Moopli**
Signed-off-by: Filip Burlacu <filip.burlacu@securekey.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -94,15 +94,19 @@ generate-test-keys:
 		frapsoft/openssl
 
 .PHONY: user-agent-start
-user-agent-start: clean user-agent-wasm-docker credential-mediator-docker generate-test-config generate-test-keys
+user-agent-start: clean user-agent-wasm-docker credential-mediator-docker did-method-cli generate-test-keys
```
Comment (sudeshrshetty) [inline Makefile:97]: We don't need `generate-test-config` anymore?
if Yes, then why that target still exists in code base?

File: scripts/check_js_intergation.sh [hunk 2]
```diff
@@ -13,7 +13,12 @@ ROOT=`pwd`
 npm -v
 echo "starting containers..."
 cd $ROOT/test/bdd/fixtures/agent-wasm
-(source .env && docker-compose down && docker-compose up --force-recreate -d)
+
+(source .env && docker-compose down --remove-orphans && docker-compose -f docker-compose.yml up --force-recreate -d)
+
+bash generate_config.sh
```
Comment (sudeshrshetty) [inline scripts/check_js_intergation.sh:19]: Why we need it only for bddtests and not for user-agent-start?

File: Makefile (hunk 1 above)
Comment (Moopli) [reply Makefile:97]: Should have removed it, fixed.

File: scripts/check_js_intergation.sh (hunk 2 above)
Comment (Moopli) [reply scripts/check_js_intergation.sh:19]: fixed.

Comment (sudeshrshetty) [reply scripts/check_js_intergation.sh:19]: not sure what did you fix here

Comment (Moopli) [reply scripts/check_js_intergation.sh:19]: I added this to scripts/user_agent_start.sh, so now user-agent-start also splits the docker-compose and generates the configs.

File: scripts/user_agent_start.sh [hunk 3]
```diff
@@ -9,4 +9,9 @@ set -e
 echo "Running $0"
 
 cd test/bdd/fixtures/agent-wasm
-(source .env && docker-compose down && docker-compose up --force-recreate)
\ No newline at end of file
+
+(source .env && docker-compose down --remove-orphans && docker-compose -f docker-compose.yml up --force-recreate -d)
+
+bash generate_config.sh
```
Comment (Moopli) [inline scripts/user_agent_start.sh:15]: @sudeshrshetty

**Pull request merged**